packages/realm_dart/lib/src/realm_class.dart

```diff
@@ -510,26 +510,9 @@ class Realm implements Finalizable {
     return realmCore.realmEquals(this, other);
   }
 
-  static Logger _logger = realmCore.defaultRealmLogger;
-
-  /// The logger to use for Realm logging in this `Isolate`
+  /// The logger to use for Realm logging in all [Isolate]s
```

Comment:
I think the `Isolate` bit is somewhat unclear here - not sure why it's relevant. I think we could remove that and instead add info on how to subscribe for messages instead. Something like:

```suggestion
  /// The logger that will emit log messages from the database and sync operations. To receive log
  /// messages, use `logger.onRecord.forEach` and pass in a callback to be invoked for every item.
```

Reply:
Okay, I'll reword a little. Don't think we should mention `forEach`, just the `Realm.logger.onRecord` stream.